cmd/earthly/account_cmds_test.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"context"
+	"testing"
+
+	"github.com/earthly/earthly/conslogging"
+	"github.com/poy/onpar"
+	"github.com/poy/onpar/expect"
+	"github.com/poy/onpar/matchers"
+)
+
+func TestAccountCmdsHelp(t *testing.T) {
+	type testCtx struct {
+		t      *testing.T
+		expect expect.Expectation
+	}
+
+	o := onpar.BeforeEach(onpar.New(t), func(t *testing.T) testCtx {
+		return testCtx{
+			t:      t,
+			expect: expect.New(t),
+		}
+	})
+	defer o.Run()
+
+	o.Spec("Help command usage does not end with '.'", func(tt testCtx) {
+		ctx := context.TODO()
+		app := newEarthlyApp(ctx, conslogging.ConsoleLogger{})
+		lines := app.accountCmds()
```

Comment:
Rather than duplicate the various `account_cmds_test.go`, `project_cmds_test.go` files, is it possible to combine them in a single test function (and file)?

Take a look at how the autocompletion works under https://github.com/earthly/earthly/blob/main/cmd/earthly/autocomplete.go#L79

it does a similar call to `app := newEarthlyApp(ctx, conslogging.ConsoleLogger{})` but then recursively traverses all the subcommands under https://github.com/earthly/earthly/blob/main/autocomplete/complete.go#L354 and https://github.com/earthly/earthly/blob/main/autocomplete/complete.go#L490

Reply:
Yeah, I can combine them all into a single file. I had been thinking that if more tests are eventually added to the cli commands then the files for each part would then already be there but it makes sense to keep them altogether for now.